docs/components/Banner/Banner.stories.mdx

```diff
@@ -30,8 +30,8 @@ width.
 #### "Global" Banner (mobile)
 
 When the message from a Banner is applicable to the entire view, it should be
```

Comment:
Is this an accurate description of a global banner? It's applicable to the entire application (hence "global"), not just the view. I believe the current positioning description is still applicable to a banner relating to a specific screen.